plans/tier1.fmf

```diff
@@ -249,3 +249,20 @@ description+: |
         discover+:
             test+<:
                 - conversion-method/rhsm_conversion
+
+    /host-metering:
+        enabled: false
+        adjust+:
+            # TODO: enable only when running on AWS, GCP, Azure
+            - enabled: true
+              when: distro == centos-7
+        discover+:
+            filter:
+            - tag:checks-after-conversion | tag:host-metering
+        prepare+:
+            - name: test host-metering
+              how: shell
+              script: pytest -svv tests/integration/*/destructive/host-metering/run_conversion.py
+            - name: reboot after conversion
+              how: ansible
+              playbook: tests/ansible_collections/roles/reboot/main.yml
```

Comment:
We actually do not use this anymore. The whole `prepare` should not be used anymore. It should look like this:
```
      discover+:
            test+<:
                - host-metering/run-host-metering-conversion
            test+:
                - host-metering/test-active-host-metering
```
Then the "tests/integration/tier1/destructive/host-metering/main.fmf" should contain this
```
/run-host-metering-conversion:
    tag+:
        - test-host-metering-conversion
    test: pytest -svv -m test_host_metering

/test-active-host-metering:
    tag+:
       - test-active-host-metering
    test: pytest -svv -m test_active_host_metering
```

Reply:
@kokesak yes that's okay

Reply:
Thanks @danmyway for trying that